ts/src/management/tunnelManagementHttpClient.ts

```diff
@@ -133,7 +136,7 @@ export class TunnelManagementHttpClient implements TunnelManagementClient {
      */
     public constructor(
         userAgents: (ProductHeaderValue | string)[] | ProductHeaderValue | string,
-        apiVersion: string,
+        apiVersion: ManagementApiVersions,
```

Comment:
Perhaps add an `apiVersion` param to the constructor signature comments above for consistency.

Reply:
good catch